modules/core/src/main/scala/gem/math/Angle.scala

```diff
@@ -4,33 +4,33 @@
 package gem.math
 
 import scalaz.{ Monoid, Show }
-import scalaz.std.anyVal.intInstance
+import scalaz.std.anyVal.longInstance
 import scalaz.syntax.equal._
 
 /**
- * Exact angles represented as integral milliarcseconds. These values form an Abelian group over
+ * Exact angles represented as integral microarcseconds. These values form an Abelian group over
  * addition, where the inverse is reflection around the 0-180° axis. The subgroup of angles where
- * integral milliarcseconds correspond with clock milliseconds (i.e., where they are evenly
- * divisible by 15 milliarcseconds) is represented by the HourAgle subtype.
- * @param toMilliarcseconds This angle in milliarcseconds. Exact.
+ * integral microarcseconds correspond with clock microseconds (i.e., where they are evenly
+ * divisible by 15 microarcseconds) is represented by the HourAgle subtype.
+ * @param toMicroarcseconds This angle in microarcseconds. Exact.
  */
-sealed class Angle protected (val toMilliarcseconds: Int) {
+sealed class Angle protected (val toMicroarcseconds: Long) {
 
   // Sanity checks … should be correct via the companion constructor.
-  assert(toMilliarcseconds >= 0, s"Invariant violated. $toMilliarcseconds is negative.")
-  assert(toMilliarcseconds < 360 * 60 * 60 * 1000, s"Invariant violated. $toMilliarcseconds is >= 360°.")
+  assert(toMicroarcseconds >= 0, s"Invariant violated. $toMicroarcseconds is negative.")
+  assert(toMicroarcseconds < 360L * 60L * 60L * 1000L * 1000L, s"Invariant violated. $toMicroarcseconds is >= 360°.")
 
   /** Flip this angle 180°. Exact, invertible. */
   def flip: Angle =
     this + Angle.Angle180
 
   /** Additive inverse of this angle (by mirroring around the 0-180 axis). Exact, invertible. */
   def unary_- : Angle =
-    Angle.fromMilliarcseconds(-toMilliarcseconds.toLong)
+    Angle.fromMicroarcseconds(-toMicroarcseconds.toLong)
 
   /** This angle in decimal degrees. Approximate, non-invertible */
   def toDoubleDegrees: Double =
-    toMilliarcseconds.toDouble / (60.0 * 60.0 * 1000.0)
+    toMicroarcseconds.toDouble / (60.0 * 60.0 * 1000.0 * 1000.0)
 
   /** This angle in decimal radisns. Approximate, non-invertible */
   def toDoubleRadians: Double =
```

Comment:
I guess that's how I read `: Double` … all bets are off once you go to floating point. I'm not opposed to renaming but I wonder what the convention should be in the codebase generally since there's a lot of floating point.